### DISCUSSION TITLE
Page de log

### DISCUSSION THREAD

**Discussion by M0rgan52** [M0rgan52/Interface_Joueur_JDR#1] [Ideas]
**Style :** 
- Trouver un logo
- Rendre plus agréable les liaisons header et footer

**Comment by M0rgan52** [comment]
Liste couleur : 
- #B88FEB
- #BA77D4
- #8777D4

**Comment by M0rgan52** [comment]
Style - couleurs et police

**Comment by CyrilD88** [comment]
Fonctionnalité - liens vers la page d'accueil
Texte - phrase d'accroche

**Comment by M0rgan52** [comment]
Texte - texte footer
Texte - titre
